src/main/java/frc/robot/Robot.java

```diff
@@ -152,7 +153,9 @@ public void testPeriodic() {
    * This function drives the robot at a certain speed.
    */
   private void driveSpeed() {
-    if (!isInControlPanelMode) {
+    if (isInControlPanelMode) {
+      differentialDrive.stopMotor();
```

Comment:
To save some future pain, might be good to add a comment here explaining that this is necessary to have here because, if not present, errors about the differential drive not being updated will be logged.

Reply:
ok

Reply:
Ok, shall do.